active/teachingtip/TeachingTipSpec.md

```diff
@@ -0,0 +1,575 @@
+# Teaching tip
+
+## Description
+
+A teaching tip is a content-rich nonmodal flyout that provides contextual information. A teaching tip looks like a flyout, except that it may target a UI element with a tail or be persistent and require the user to manually dismiss it. A teaching tip is often used for informing, reminding, and teaching users about important and new features that may enhance their experience. 
```

Comment:
```suggestion
A teaching tip is a semi-persistent and content-rich flyout that provides contextual information. It is often used for informing, reminding, and teaching users about important and new features that may enhance their experience. 
```

Reply:
@MikeHillberg Does this look better? It is now ~30% shorter than Password Box's opening paragraph. I'm happy to revise it if you think it is more critical to share the pointing/non-pointing and light dis-miss features here as opposed to how it should be used.

Reply:
```suggestion
A teaching tip is a semi-persistent and content-rich flyout that provides contextual information. It is often used for informing, reminding, and teaching users about important and new features that may enhance their experience.
```